src/sentry/api/endpoints/team_release_count.py

```diff
@@ -0,0 +1,37 @@
+from collections import defaultdict
+
+from django.db.models import Count
+from django.db.models.functions import TruncDay
+from rest_framework.response import Response
+
+from sentry.api.base import EnvironmentMixin
+from sentry.api.bases.team import TeamEndpoint
+from sentry.api.utils import get_date_range_from_params
+from sentry.models import Project, Release
+
+
+class TeamReleaseCountEndpoint(TeamEndpoint, EnvironmentMixin):
+    def get(self, request, team):
+        """
+        Returns a dict of team projects, and a time-series list of release counts for each.
+        """
+        project_list = Project.objects.get_for_team_ids(team_ids=[team.id])
+        start, end = get_date_range_from_params(request.GET)
+
+        bucketed_releases = (
+            Release.objects.filter(
+                projects__in=project_list, date_added__gte=start, date_added__lte=end
+            )
+            .distinct()
```

Comment:
Using `distinct` can be expensive because postgres ends up having to sort the data to de-dupe it. Another option is to replace `projects__in=project_list` with `id__in=ReleaseProject.objects.filter(project__in=project_list).values("release_id")`

Actually, looking at this do you want to have distinct release rows here?

Reply:
I added distinct because it seemed to be double counting releases that were in multiple projects which seemed undesirable...can look into replacing `projects__in` though

Reply:
Should we remove this distinct now?

Reply:
....yes 😬